### PR TITLE
Remove env vars refs

### LINKED ISSUE #22: Update environment variable prefixes
This should just be a pass to rename environment variables to have the `APPTAINER_` and `APPTAINERENV_` prefixes

### PR DESCRIPTION
**Pull request by pedroalvesbatista**
## Description of the Pull Request (PR):

Updates references from SINGULARITY to APPTAINER.

## This fixes or addresses the following GitHub issues:

- Fixes #22

### CODE REVIEW

**Review by DrDaveD — CHANGES_REQUESTED** [review]
This seems to include too many commits, some not relevant to the referenced issue.

File: appendix.rst [hunk 1]
```diff
@@ -20,294 +20,295 @@ below with their respective functionality.
...
    filesystems from your host. Default is set to false.
 
-#. **SINGULARITY_CONTAINALL**: To contain not only file systems, but
+#. **APPTAINER_CONTAINALL**: To contain not only file systems, but
    also PID, IPC, and environment. Default is set to false.
 
-#. **SINGULARITY_CONTAINLIBS**: Used to specify a string of file names
-   (comma separated string) to bind to the ``/.singularity.d/libs``
+#. **APPTAINER_CONTAINLIBS**: Used to specify a string of file names
+   (comma separated string) to bind to the ``/.apptainer.d/libs``
```
Comment (ikaneshiro) [inline appendix.rst:65]:
```suggestion
   (comma separated string) to bind to the ``/.singularity.d/libs``
```
This should remain the same

File: appendix.rst [hunk 2]
```diff
@@ -512,7 +513,7 @@ Notes
 When bootstrapping from a Singularity Hub image, all previous definition
 files that led to the creation of the current image will be stored in a
 directory within the container called
-``/.singularity.d/bootstrap_history``. {Singularity} will also alert you
+``/.apptainer.d/bootstrap_history``. {Singularity} will also alert you
```
Comment (ikaneshiro) [inline appendix.rst:516]:
```suggestion
``/.singularity.d/bootstrap_history``. {Singularity} will also alert you
```

File: appendix.rst [hunk 3]
```diff
@@ -908,7 +909,7 @@ currently residing in docker's daemon internal storage:
    REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
    alpine              latest              965ea09ff2eb        7 weeks ago         5.55MB
 
-   $ singularity run docker-daemon:alpine:latest
+   $ apptainer run docker-daemon:alpine:latest
```
Comment (ikaneshiro) [inline appendix.rst:912]: Lets not change these command docs until we are doing a PR for #17

File: appendix.rst [hunk 4]
```diff
@@ -927,7 +928,7 @@ a docker image stored in a ``docker-save`` formatted tar file:
 
    $ docker save -o alpine.tar alpine:latest
 
-   $ singularity run docker-archive:$(pwd)/alpine.tar
+   $ apptainer run docker-archive:$(pwd)/alpine.tar
```
Comment (ikaneshiro) [inline appendix.rst:931]: same here

**Review by ikaneshiro — APPROVED** [review]

**Pull request merged**